src/test/java/org/mule/test/http/functional/HttpRequesterTlsRestrictedProtocolsAndCiphersTestCase.java

```diff
@@ -21,4 +21,9 @@
   protected String getConfigFile() {
     return "http-request-restricted-protocols-ciphers-config.xml";
   }
+
+  @Override
+  protected boolean doTestClassInjection() {
+    return true;
```

Comment:
Is not this the default value on the functional tests?

Reply:
The default is set for `MuleArtifactFunctionalTestCase`, this one extends from another test class (there is a TODO to change it)